src/lang/extra/korean.txt

```diff
@@ -1,18 +1,43 @@
+##override on
+STR_NEWS_VEHICLE_AUTORENEW_FAILED                               :{WHITE}{VEHICLE}의 자동 교체에 실패하였습니다{}{STRING}
+STR_NEWGRF_ERROR_FATAL_POPUP                                    :{WHITE}NewGRF "{STRING}"에서 치명적인 오류가 발생했습니다:{}{STRING}
+STR_NEWGRF_ERROR_POPUP                                          :{WHITE}NewGRF  "{STRING}"에서 오류가 발생했습니다:{}{STRING}
+STR_ORDER_TEXT                                                  :{STRING} {STRING} {STRING}
+STR_TIMETABLE_TRAVEL_FOR                                        :{STRING}에 걸쳐 운행
+STR_TIMETABLE_TRAVEL_FOR_SPEED                                  :{0:STRING} 동안 최대 {1:VELOCITY}의 속력으로 운행
+STR_TIMETABLE_TRAVEL_FOR_ESTIMATED                              :운행 ({STRING} 동안, 시간표 작성 안 됨)
+STR_TIMETABLE_TRAVEL_FOR_SPEED_ESTIMATED                        :최대 {1:VELOCITY}의 속력으로 운행 ({0:STRING} 동안, 시간표 작성 안 됨)
+STR_TIMETABLE_STAY_FOR_ESTIMATED                                :({STRING} 동안 정차, 시간표 작성 안 됨)
+STR_TIMETABLE_AND_TRAVEL_FOR_ESTIMATED                          :({STRING} 동안 운행, 시간표 작성 안 됨)
+STR_TIMETABLE_STAY_FOR                                          :& {STRING} 동안 정차
+STR_TIMETABLE_AND_TRAVEL_FOR                                    :& 다음 목적지({STRING})로 이동
+STR_TIMETABLE_TOTAL_TIME                                        :{BLACK}이 시간표는 완주하는데 {STRING}이 걸릴 것입니다.
+STR_TIMETABLE_TOTAL_TIME_INCOMPLETE                             :{BLACK}이 시간표를 완주하는데 최소 {STRING}이 필요합니다 (시간표가 일부만 지정됨)
+STR_TIMETABLE_STATUS_LATE                                       :{BLACK}이 차량은 현재 {STRING} 늦게 운행하고 있습니다
+STR_TIMETABLE_STATUS_EARLY                                      :{BLACK}이 차량은 현재 {STRING} 빨리 운행하고 있습니다
+##override off
```

Comment:
Checks are failing because "overrides are only allowed in the base translation" (`english.txt`), not in other translations.